lib/utils/uacc.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
Shouldn't this be `uacc.c`? Also I don't think this needs to reside in `utils`. I'd either put it as a subpackage of `srv` or a top level package in `lib` with a nice Go API that can be used in `reexec.go`.

Reply:
I've moved it out in some commits I'll push in a second. The reason for naming it .h is .c usually intends for the logic in the file to be exposed in the whole binary but we just want to access it in the Go file so I made the functions static and simply included it. .h is used here as it's intended to be a single use header-only style module.